main/src/ca/uwaterloo/flix/language/phase/unification/KindUnification.scala

```diff
@@ -48,6 +48,10 @@ object KindUnification {
     case (Kind.CaseSet(sym), Kind.WildCaseSet) =>
       Some(Kind.CaseSet(sym))
 
+    case (Kind.Error, k) => Some(k)
+
```

Comment:
extra linebreak

Reply:
It was on purpose. The other cases are similarly separated :)